usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_category.md

```diff
@@ -6,16 +6,13 @@
 These endpoints return data that is grouped in preset units to support the various data visualizations on USAspending.gov's Advanced Search page.
 ### Request
 
-filters (**REQUIRED**): The filters to find with said category. The filter object is defined here: [Filter Object](../search_filters.md). 
+filters (**REQUIRED**): The filters to find with said category. The filter object is defined here: [Filter Object](../search_filters.md).
 
 category (**REQUIRED**): String value. Parameter indicating which category to aggregate and return. Below are the following categories available.
 
 * `awarding_agency`
 * `awarding_subagency`
-* `funding_agency`
```

Comment:
Why are the funding agency/subagency categories being removed from the docs?

Reply:
Funding Agency and Subagency are not necessary categories, there are concerns that the fields are too sparse for useful analysis. If it wasn't this late in the sprint I would modify the API contract and remove them from the code.